hil/client/user.py

```diff
@@ -13,21 +13,17 @@ class User(ClientBase):
     manipulate users related objects and relations.
     """
 
-    def create(self, username, password, privilege):
+    def create(self, username, password, is_admin):
         """Create a user <username> with password <password>.
 
-        <privilege> may by either "admin" or "regular",
+        <is_admin> may by either true or false,
```

Comment:
might just say `<is_admin> is a boolean`

Reply:
👍